chimerapy/manager.py

```diff
@@ -236,7 +191,7 @@ def save_meta(self):
 
         # Generate meta record
         meta = {
-            "workers": list(self.workers.keys()),
+            "workers": list(self.state.workers.keys()),
             "nodes": list(self.graph.G.nodes()),
             "worker_graph_map": self.worker_graph_map,
             "nodes_server_table": self.nodes_server_table,
```

Comment:
ToDo: This can be an enhancement where in we incorporate these later on to the state as well.

Reply:
Agreed